File: UserServer/src/main/java/cc/nobrain/dev/userserver/security/PasswordConfig.java
package cc.nobrain.dev.userserver.security;

import lombok.Getter;
import org.springframework.context.annotation.Bean;
import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
import org.springframework.security.crypto.password.PasswordEncoder;

@Getter
public class PasswordConfig {
    public static final PasswordEncoder passwordEncoder = new BCryptPasswordEncoder(10);
}

File: UserServer/src/main/kotlin/cc/nobrain/dev/userserver/common/security/PasswordConfig.kt
package cc.nobrain.dev.userserver.common.security;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Profile;
import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
import org.springframework.security.crypto.password.PasswordEncoder;

@Configuration
@Profile("!test")
public class PasswordConfig {
    @Bean
    fun passwordEncoder(): PasswordEncoder {
        return BCryptPasswordEncoder(10);
    }
}
